code/src/java/gmgen/util/MiscUtilities.java

```diff
@@ -24,19 +22,13 @@
 {
 	private MiscUtilities()
 	{
-		//Do not allow instantiation of utility class
```

Comment:
Not sure if this is being flagged by another tool as useless, but this comment is actually used to suppress either CheckStyle or PMD (I forget which) complaining about empty methods